samples/drivers/espi/boards/mec172xmodular_assy6930.conf

```diff
@@ -0,0 +1,10 @@
+# eSPI + mec172xevb_assy6906
```

Comment:
This new addition to the eSPI samples should be split into a separate commit.

Reply:
Moved the config/overlay files into separate commit